app/javascript/controllers/help_how_controller.js

```diff
@@ -0,0 +1,20 @@
+import { Controller } from '@hotwired/stimulus'
+
+export default class extends Controller {
+  static targets = ['frame']
+
+  connect () {
+    this.origSource = this.frameTarget.src
+    // Dynamically change the help turbo-frame source to show "Request access to another collection" by default.
+    this.element.addEventListener('show.bs.modal', event => {
+      const button = event.relatedTarget
+      if (!button) return
+      const showCollections = button.getAttribute('data-bs-showcollections') === 'true'
+      if (showCollections) {
+        this.frameTarget.src = this.origSource + '&show_collections=true'
+      } else {
+        this.frameTarget.src = this.origSource
+      }
+    })
+  }
+}
```

Comment:
this hooks into the bootstap modal opening event and adjust the turbo frame source if it sees the specific attribute we set for pre-selecting request a collection